cmd/backup-manager/app/clean/clean.go

```diff
@@ -67,14 +68,14 @@ func (bo *Options) cleanRemoteBackupData(bucket string, opts []string) error {
 	destBucket := util.NormalizeBucketURI(bucket)
 	args := util.ConstructArgs(constants.RcloneConfigArg, opts, "deletefile", destBucket, "")
 	output, err := exec.Command("rclone", args...).CombinedOutput()
-	if err != nil {
-		if exitError, ok := err.(*exec.ExitError); ok {
-			if code := exitError.ExitCode(); code == 3 || code == 4 {
-				klog.Infof("cluster %s backup %s has already been deleted before", bo, bucket)
-				return nil
-			}
-		}
-		return fmt.Errorf("cluster %s, execute rclone deletefile command failed, output: %s, err: %v", bo, string(output), err)
+	if err != nil && !strings.Contains(string(output), "doesn't exist") {
+		return fmt.Errorf("cluster %s, execute rclone deletefile command to delete archive failed, output: %s, err: %v", bo, string(output), err)
```

Comment:
why changing this back?

Reply:
Because according to test, this error code is not acurate. When `rclone` returns `dir doesn't exist` error the exit code is 1. May the last time I forgot to use `hack/local-up-operator.sh` to update operator.

Reply:
maybe `rclone delete` is a better command in this scenario

https://rclone.org/commands/rclone_delete/

in my testing, it succeeds if not file found at the specified path

Reply:
Yes. It won't even return error. Should we use this command?
C.C. @DanielZhangQD

Reply:
It's OK if the file can be removed if exist.

Reply:
Revise to `rclone delete`